clr/commands.py

```diff
@@ -11,14 +11,11 @@
 import time
 from collections import namedtuple
 from typing import Dict, Callable, Any
+import argparse
 
 import clr.config
 
-# Sentinal for arg defaults in get_command_spec to indicate there is no default.
-# Because we are pickling command specs for clr cache, use a random int and
-# check for equality rather than object identity.
-# TODO(michael.cusack): Move command spec to inspect.Signature and remove this.
-NO_DEFAULT = 4194921784511160246
+CommandSpec = namedtuple('CommandSpec', 'docstr signature')
```

Comment:
I realize `collections.namedtuple` is already being used in this file, but any reason to not instead favor `typing.NamedTuple` or `@dataclass`? Both are a bit more explicit about their members and also include typing.